### ISSUE TITLE
screen is black, no errors

### ISSUE THREAD

**Issue by ston1fam** [MetaShrimps/screenshare#1]
renamed script in ServerScriptService in Script, Server, FrameStackerServer and same names with .lua
playit agent is working, TCP tunnel. Screen in Workspace
![image](https://github.com/MetaShrimps/screenshare/assets/132093759/51d42a26-c526-49bd-9541-6884fd169bcf)
![image](https://github.com/MetaShrimps/screenshare/assets/132093759/388e2715-0363-40ce-a645-dbcc8fbf1908)
![image](https://github.com/MetaShrimps/screenshare/assets/132093759/4d355be6-2a5c-435d-ac98-3ef7f6593751)
![image](https://github.com/MetaShrimps/screenshare/assets/132093759/8d1aef55-bdbe-46f9-8e68-8d22cbe3ae69)

**Comment by baldi2** [comment]
yeah it's something wrong with canvas draw, it's happening to me too

**Comment by KHGman** [comment]
forgot to put 2 dots on the link likee this : "https://viladplayitlink:" .. PORT)
                                                                           dots here->    !    over the explanation mark

**Comment by ghost** [comment]
in line 5 there is a link https://www.roblox.com/library/13252615540/CanvasDraw-Edited get this im not sure where to put it im still trying to find out

**Comment by DevGeah** [comment]
> in line 5 there is a link https://www.roblox.com/library/13252615540/CanvasDraw-Edited get this im not sure where to put it im still trying to find out

I tried putting in in server script service, nothing happened. Then I tried putting it inside of the main lua script, and I got a whole bunch of errors saying, "task.desynchronize() should only be called from a script that is a descendant of an Actor"

**Comment by WhundAlt** [comment]
just delete all the task.desynchronize() and the task.synchronize() from the script and it will work, atleast it worked for me
